src/operator/account/CancelAccountOverlay.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {FC, useContext} from 'react'
+import {
+  Overlay,
+  Gradients,
+  Alert,
+  ComponentColor,
+  IconFont,
+  ButtonBase,
+  ButtonShape,
+} from '@influxdata/clockface'
+import {AccountContext} from 'src/operator/context/account'
+
+const CancelAccountOverlay: FC = () => {
+  const {
+    account,
+    handleDeleteAccount,
+    setCancelOverlayVisible,
+    cancelOverlayVisible,
+  } = useContext(AccountContext)
+
+  const cancelAccount = () => {
+    if (account?.cancellable) {
+      try {
+        handleDeleteAccount()
+      } catch (e) {
+        setCancelOverlayVisible(false)
+      }
+    }
+  }
+
+  const message = `
+    This action will permanently cancel the Account
+    ${account?.id ?? 'N/A'} and all of its organizations. 
+    Users that belong to multiple accounts are still able 
+    to access their other accounts.`
+  return (
+    <Overlay
+      visible={cancelOverlayVisible}
+      renderMaskElement={() => (
+        <Overlay.Mask gradient={Gradients.DangerDark} style={{opacity: 0.5}} />
+      )}
+      testID="cancel-overlay"
+      transitionDuration={0}
+    >
+      <Overlay.Container maxWidth={600}>
+        <Overlay.Header
+          title="Cancel Account"
+          style={{color: '#FFFFFF'}}
+          onDismiss={() => setCancelOverlayVisible(!cancelOverlayVisible)}
+        />
```

Comment:
Same comment here - I think dismissing should just set the cancel overlay to false rather than flipping a toggle.